mix.exs

```diff
@@ -91,14 +91,13 @@ defmodule Trento.MixProject do
   defp aliases do
     [
       start: [
-        "cmd docker-compose up -d",
-        "deps.get",
-        "ecto.create",
-        "ecto.migrate",
-        "event_store.setup",
+        "install",
+        "setup",
         "phx.server"
       ],
-      setup: ["deps.get", "event_store.setup", "ecto.setup", "init_grafana_dashboards"],
+      install: ["deps.get", "cmd --cd assets npm install"],
```

Comment:
I'm not sure the team wants `npm` to be part of the mixfile, I think we should vote on this :sweat_smile:

Reply:
Hmm, I thought it was okay since it's already there for the `assets.deploy` command.

If we don't want npm there, what's the highest order task manager? I understand we don't want a Makefile.

Reply:
You're right, keep it that way.

Reply:
+1 to keep npm install, mix should also take care of this kind of stuff